bin/htrace.sh

```diff
@@ -39,6 +39,18 @@
 __init_params=()
 __script_params=("$@")
 
+if [[ "$(uname)" == "Darwin" ]] ; then
+
+  [ ! -z "$(brew --prefix)" ] && PATH=$(brew --prefix)/opt/gnu-getopt/bin:$(brew --prefix)/opt/gnu-sed/libexec/gnubin:$(brew --prefix)/opt/coreutils/libexec/gnubin:$PATH
+  [ ! -z "$(composer global config bin-dir --absolute 2>/dev/null)" ] && PATH=$(composer global config bin-dir --absolute 2>/dev/null):$PATH
+
+# shellcheck disable=SC2003,SC2046
+elif [[ "$(expr substr $(uname -s) 1 5)" == "Linux" ]] ; then
```

Comment:
why is there a substr necessary? I only ever see `uname -s` return "Linux" on Linux.

also, if there's an `else` that always returns `true` anyways .. what`s the point?

I kind of dislike disabling `shellcheck` strong .. (ditto in `setup.sh`)

Reply:
1) I used a better solution for this
2) For the future
3) Yes, you're right, I'll correct it